src/main/java/lotto/controller/LottoApplicationController.java

```diff
@@ -0,0 +1,64 @@
+package lotto.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lotto.domain.Lotto;
+import lotto.domain.LottoNumberFactory;
+import lotto.domain.LottoNumbers;
+import lotto.domain.LottoStatistics;
+import lotto.domain.PurchaseAmount;
+import lotto.domain.WinningNumbers;
+
+public class LottoApplicationController {
+	private static final int PURCHASE_FINISH = 0;
+	private PurchaseAmount purchaseAmount;
+	private WinningNumbers winningNumbers;
+	private LottoStatistics lottoStatistics = new LottoStatistics();
+	private List<Lotto> lottos = new ArrayList<>();
```

Comment:
```suggestion
	private Lottos lottos;
```
Lotto에 대한 일급 컬렉션을 만들어보는 건 어떨까요 ?

추가로 코멘트 드려보자면 전반적으로 클래스 내부에 다른 클래스에 대한 참조가 있을 때 , 
미리 초기화를 해주고 난 후에 상태를 변경하는 방식으로 구현되어 있는데요. 
불변 객체로 사용하기 위한 리팩터링을 해보시는건 어떨까요 !? :)

https://velog.io/@conatuseus/Java-Immutable-Object%EB%B6%88%EB%B3%80%EA%B0%9D%EC%B2%B4